packages/rocketchat-lib/server/functions/saveUser.js

```diff
@@ -94,6 +94,48 @@ function validateUserData(userId, userData) {
 	}
 }
 
+function validateUserEditing(userId, userData) {
+	const editingMyself = userData._id && userId !== userData._id;
```

Comment:
The const name is misleading. Shouldn't it be `editingOtherUser`? Otherwise all the conditionals are wrong.

Reply:
oops.